site_scons/site_tools/unittest.py

```diff
@@ -91,14 +92,14 @@ def __call__(self, target, source, env):
                 shutil.move(out, out + '.failed')
                 msg = '*** Unit test failed, check log file ' + out + '.failed ***'
                 sep = '*' * len(msg)
-                print sep + '\n' + msg + '\n' + sep
+                print(sep + '\n' + msg + '\n' + sep)
```

Comment:
Should we modernize further:

    print(sep, msg, sep, sep='\n')

Mixing `sep` as argument and keyword argument is confusing, I'd rename `sep` variable to something else